src/Pages/Profile/MemberView/ChangePassword.js

```diff
@@ -5,6 +5,7 @@ export default function ChangePasswordModal(props) {
   const { bannerCallback = (message, color) => { }, confirmClassAddons } = props;
   const [password, setPassword] = useState('');
   const [confirmPassword, setConfirmPassword] = useState('');
+  const [showPassword, setShowPassword] = useState(false);
```

Comment:
by default we should show it?

Reply:
I think its usually hidden by default on most sites I remember